rotkehlchen/tests/unit/globaldb/test_globaldb.py

```diff
@@ -988,52 +988,52 @@ def test_general_cache(globaldb):
     ts_test_start = ts_now()
     with globaldb.conn.write_ctx() as write_cursor:
         # write some values
-        globaldb_set_general_cache_values(
+        globaldb_set_cache_values(
             write_cursor=write_cursor,
-            key_parts=[GeneralCacheType.CURVE_LP_TOKENS],
+            key_parts=[GeneralCacheType.CURVE_POOL_TOKENS],
```

Comment:
Why this change in the cache type key?

Reply:
This is not answered

Reply:
Oww i said reason is;

Since the db has been updated to include a unique_cache depending on the key, I changed the key type to make sure it is one of those which sends data to the general_cache table